WEP0000_wep-proposal.rst

```diff
@@ -0,0 +1,228 @@
+
+For example, if this WEP is accepted, it means the system is now in place and the CoreOps team will publish the information to the community website and establish the WEP team, along with other related tasks that need to be done to implement the process in the community. 
+
+Some WEPs might require specific community segments to implement the resolution, and such requirements are described in the WEPs directly. 
+
+WEP format and style
+--------------------
+
+To make the authoring and reviewing process easier, WEPs must follow a common format and style. 
+
+We use `reStructuredText <http://docutils.sourceforge.net/rst.html>`_ markup language for all WEPs, to align with the same markup that is used in all other Write the Doc documentation.
+
+In most cases, you can start by copying the `WEP template <../wep-template.rst>`_ and adding content as needed. The following sections describe the various sections of the template:
+
+.. AI - Make the template!
+
+File name
+.........
+The file name must follow the naming convention ``WEP[####]_[descriptive-but-short-title]``. For example, this WEP file is named ``WEP0000_wep-proposal``. In another example, a WEP to propose a new Slack channel naming convention might be named ``WEP0001_slack-channel-renaming``.
+  
+Metadata
+........
+The WEP file must begin with a header list of metadata fields, using the standard `RST field list <http://docutils.sourceforge.net/docs/ref/rst/restructuredtext.html#field-lists>`_ format.
+
+The field list usually includes the following fields:
+
+- ``:WEP-Number:`` - Number of the WEP, in running order in 4-digit format. Only use the number value, without the acronym **WEP**. For example, this WEP is numbered ``0000``. If the number is unknown or unassigned when you first create the WEP, you can leave this field blank and the number will be updated at the resolution stage.
+- ``:Created:`` Date of initial creation, in the format YYYY-MM-DD. 
+- ``:Last-Modified:`` Date of the most recent changes to the WEP, in the format YYYY-MM-DD. 
+- ``:Author:`` - Full names of one or more authors of the WEP. Do not write email addresses or social media nicknames in this field. If the WEP is co-authored by multiple people, use a bulleted list similar to the header of this WEP.
+- ``:Status:`` - Current status of the WEP. All WEPs begin with a ``Draft`` status, and the field changes with every stage in the WEP lifecycle. For example, ``Accepted``, ``Rejected``, ``Withdrawn``, ``Final``, or ``Superseded``.
+- ``:Replaces:`` - Optional. If this WEP was created instead of a previous WEP that was rejected or withdrawn, list the number and title of the previous WEP.
+- ``:Superceded-By:`` - Optional. If this WEP  rejected or withdrawn and a newer WEP was submitted instead with updated content, list the number and title of the WEP that supercedes this WEP.
```

Comment:
```suggestion
- ``:Superceded-By:`` - Optional. If this WEP rejected or withdrawn and a newer WEP was submitted instead with updated content, list the number and title of the WEP that supercedes this WEP.
```

strong eyes @scottydocs !

Reply:
“If this WEP rejected” should be “If this WEP was rejected”

Reply:
Thanks, fixed.